### PR TITLE
Feat color tracking from camera

### PR DESCRIPTION
**Pull request by mrzilvis**
Review my code.

### CODE REVIEW

File: TOPBanga/VideoFromFile.cs [hunk 1]
```diff
@@ -83,6 +88,32 @@ private void DetectionButton_Click(object sender, EventArgs e)
                     this.Picture.Image = currentImage.Bitmap;
             });
             this.videoTickTimer.Start();
+            
+        }
+
+        private void button1_Click(object sender, EventArgs e)
```
Comment (pauliuskalv) [inline TOPBanga/VideoFromFile.cs]: Rename the button button1 to an appropriate name

**Comment by pauliuskalv** [comment]
Also, you forgot to push the designer code, so button1 doesn't exist

**Comment by pauliuskalv** [comment]
Bugs are present in this branch, so we can't merge yet

**Comment by solarbeam** [comment]
I would suggest more fixes before merging this; most combinations of button presses crash the program.

File: TOPBanga/VideoFromFile.cs [hunk 2]
```diff
@@ -37,7 +38,12 @@ private void BrowseButton_Click(object sender, EventArgs e)
             openFileDialog.Filter = "MP4 file|*.mp4";
             if (openFileDialog.ShowDialog() == DialogResult.OK)
             {
+                if (this.webcamOn){
```
Comment (solarbeam) [inline TOPBanga/VideoFromFile.cs]: This if is not necessary.

File: TOPBanga/VideoFromFile.cs [hunk 3]
```diff
@@ -83,6 +91,32 @@ private void DetectionButton_Click(object sender, EventArgs e)
...
+
+        private void switchCam_Click(object sender, EventArgs e)
+        {
+            this.videoTickTimer.Interval = 80;
+            if (this.videoLoaded)
+            {
+                this.videoTickTimer.Stop();
+                this.videoLoaded = false;
+            }
+            this.webcamOn = true;
```
Comment (solarbeam) [inline TOPBanga/VideoFromFile.cs]: This line is potentially dangerous

File: TOPBanga/VideoFromFile.cs [hunk 4]
```diff
@@ -83,6 +91,32 @@ private void DetectionButton_Click(object sender, EventArgs e)
...
+               this.webcamOn = true;
+               this.webcam = new VideoCapture(0); //0 is default camera
+            }
+            this.currentFrame = this.webcam.QueryFrame();
+            this.Picture.Image = this.currentFrame.Bitmap;
+            Image<Bgr, byte> currentImage = this.currentFrame.ToImage<Bgr, byte>();
+            this.detector.image = currentImage;
+        }
+
+        private void ColorBox_Click(object sender, EventArgs e)
```
Comment (solarbeam) [inline TOPBanga/VideoFromFile.cs]: Why is this empty?

**Comment by solarbeam** [comment]
This request got bugged

**Pull request closed without merging**